front/package.json

```diff
@@ -207,6 +207,7 @@
     "zod-validation-error": "^3.4.0"
   },
   "devDependencies": {
+    "read-pkg-up": "^11.0.0",
```

Comment:
adding this for the test `tiptap.test.ts`, it allows to read the content of a node module at runtime